packages/components/src/components/alert/test/html.mock.ts

```diff
@@ -66,8 +86,8 @@ export const getAlertHtml = (props: AlertProps, innerHTML = '', additionalHTML =
 						_hideLabel
 						_icons={{
 							left: {
-								icon: 'codicon codicon-close',
-							},
+								icon: 'codicon codicon-close'
+							}
```

Comment:
Wozu sind diese Änderungen gut? Auch wenn es hier anscheinend keine Anwendung findet, entsprach der vorherige Style unserer Prettier-Konfiguration.

Reply:
Da die Kommas keinen Nutzen haben, können sie auch weggelassen werden.
Ich kann sie aber auch drin lassen.

Reply:
Es ist dann eine reine Code-Style-Frage. Ich würde im allgemeinen anlasslose Code-Style-Änderungen vermeiden, weil sie unnötig Änderungen erzeugen und der Code so ggf. von verschiedenen Personen hin-und-her geändert wird.

Ist in dem Fall wirklich eine Kleinigkeit, nichts weswegen ich ein PR ablehnen würde. Ich wollte es nur einmal allgemein angesprochen haben. Wenn du sowieso am Branch arbeitest, könntest du die Änderung kurz reverten.